### ISSUE TITLE
Accessibility check fail on pagination 

### ISSUE THREAD

**Issue by digitala11y** [studiopress/genesis-sample#326]
Hello,
In genesis sample the pagination links on the home page fail automated accessibility checks. This is due to providing role=navigation on the <ul> element. The navigation role is supressing the symantics of the native HTML element. the role=navigation need to be moved to a div tag above the <ul> element. Here is the code from my site
<ul role="navigation" aria-label="Pagination"><li class="active"><a href="https://www.digitala11y.com/" aria-label="Current page" aria-current="page"><span class="screen-reader-text">Go to page</span> 1</a></li>
<li><a href="https://www.digitala11y.com/page/2/"><span class="screen-reader-text">Go to page</span> 2</a></li>
<li><a href="https://www.digitala11y.com/page/3/"><span class="screen-reader-text">Go to page</span> 3</a></li>
<li class="pagination-omission"><span class="screen-reader-text">Interim pages omitted</span> …</li> 
<li><a href="https://www.digitala11y.com/page/15/"><span class="screen-reader-text">Go to page</span> 15</a></li>
<li class="pagination-next"><a href="https://www.digitala11y.com/page/2/"><span class="screen-reader-text">Go to</span> Next Page »</a></li>
</ul>

I am a blind person and a A11Y specialist so please let me know if any morre info is required from my end.

**Comment by dreamwhisper** [comment]
@digitala11y Thanks for the helpful report! 

I've verified and moved this issue to the Genesis parent theme repo since the markup for the pagination is handled in the parent theme.
